spec/spec_helper.rb

```diff
@@ -112,5 +112,15 @@
     :allow_unused_http_interactions => false
   }
 
+  secrets = Rails.application.secrets
+
+  secrets.keys.each do |provider|
+    next if [:secret_key_base, :secret_token].include?(provider) # Defaults
+    cred_hash = secrets.send(provider.to_sym)
+    cred_hash.each do |key, value|
+      c.filter_sensitive_data("<#{key.upcase}>"){ value }
```

Comment:
I'm wondering if the key should include the provider...something like `"<#{provider.upcase}_#{key.upcase}>"`...so it would show `"AZURE_CLIENT_ID"`